js/glowingbear.js

```diff
@@ -761,6 +761,26 @@ weechat.controller('WeechatCtrl', ['$rootScope', '$scope', '$store', '$timeout',
         }
     };
 
+    $rootScope.supports_formatting_date =
+        function toLocaleDateStringSupportsLocales() {
+        // toLocaleDateStringSupportsLocales taken from MDN:
+        // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference
+        // /Global_Objects/Date/toLocaleDateString
+        // #Checking_for_support_for_locales_and_options_arguments
```

Comment:
Please don't break lines in URLs, it took me two reads to recognize it. Long lines for long URLs :)